content/second-order-logic/syntax-and-semantics/inf-count.tex

```diff
@@ -60,9 +60,9 @@
 \[
 m_0, m_1, m_2, \dots
 \]
-of its !!{element}s (without repetitions).  Such an enumeration exists
+of its !!{element}s (possibly with repetitions).  Such an enumeration exists
```

Comment:
I don't think we can allow repetitions since if we do, f will not be well-defined (eg if  m_k = m_j but m_k+1 =/= m_j+1).

Reply:
Right, thanks for spotting that. I added the possibility of the enumeration to be finite. To me the notation m_0, m_1, m_2,  \dots indicates that the enumeration is infinite, but maybe that's just me.